src/components/Table.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import './Table.css';
+import { Icon } from '@iconify/react';
+import externalLinkOutline from '@iconify-icons/eva/external-link-outline';
+
+const Row = ({ location }) => {
+  return (
+    <tr className="tableListing">
+      <td className="tableThumbnail">
+        {location.thumbnail ? (
+          <img src={location.thumbnail.source} alt="location thumbnail" />
+        ) : (
+          ''
+        )}
+      </td>
+      <details>
```

Comment:
So...this is invalid HTML 😬 

A `<tr>` can only have `<td>` or `<th>` as a direct child. What I would suggest is creating a `<td>` and putting your `<details>` element inside of it.

Let me know if you need additional direction on this! Happy to throw together a gist or something!